__docs__/wonder-blocks-theming/use-styles.mdx

```diff
@@ -0,0 +1,67 @@
+import {Meta, Canvas} from "@storybook/blocks";
+
+import {
+    tokens,
+    useScopedTheme,
+    useStyles,
+} from "@khanacademy/wonder-blocks-theming";
+
+import {UseStylesExample} from "./theme-examples";
+
+<Meta title="Theming / useStyles" />
+
+# `useStyles`
+
+`useStyles` is a hook that that applies styles based on a given theme.
+
```

Comment:
In the PR description, you wrote
> The difference with useScopedTheme is that this hook will return the styles for the component, while useScopedTheme will return the theme object.

Can you add that as a note here in the docs too?